src/elements/Icon/Icon.d.ts

```diff
@@ -45,7 +45,7 @@ export interface IconProps {
   loading?: boolean;
 
   /** Name of the icon. */
-  name?: string;
+  name?: SemanticICONNAMES;
```

Comment:
Can we rename it to `SemanticICONS`? It will have more parity with `SemanticCOLORS`, `SemanticWIDTHS`, etc.

Reply:
yeah sure